<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBook</class>
 <widget class="QDialog" name="FindBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find book</string>
  </property>
  <widget class="QLabel" name="BookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Book Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBookName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Book</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
